source/brailleDisplayDrivers/hidBrailleStandard.py

```diff
@@ -126,8 +138,10 @@ def __init__(self, port="auto"):
 		self._keysDown = set()
 		self._ignoreKeyReleases = False
 
-	def _findCellValueCaps(self) -> Optional[hidpi.HIDP_VALUE_CAPS]:
-		for valueCaps in self._dev.outputValueCaps:
+	def _findCellValueCaps(self) -> List[hidpi.HIDP_VALUE_CAPS]:
```

Comment:
[Using `List` is now deprecated](https://docs.python.org/3/library/typing.html#typing.List)

```suggestion
	def _findCellValueCaps(self) -> list[hidpi.HIDP_VALUE_CAPS]:
```